talks/abstracts/CollabW2022.md

```diff
@@ -0,0 +1,78 @@
+# SSI Collaborations Workshop 2020 - Mini Workshop
+
+- **Where:** Online! :woman_technologist:
+- **When:** 4th - 7th April 2022
+- **Conference website:** <https://software.ac.uk/cw22>
+
+---
+
+## Name / Email
+
+Sarah Gibson / drsarahlgibson@gmail.com
+
+## Home Institution
+
+The International Interactive Computing Collaboration (2i2c)
+
+## Session Collaborators
+
+_Please include First Name, Surname, Home institution and email address for any additional collaborators on this session proposal, each on a separate line._
+
+- Meag Doherty, NIH
+- Min Ragan-Kelley, Simula
+- Achintya Rao, The Alan Turing Institute
+
+## Mini-workshop or demo session title
+
+Crowdsourcing Community Practices for Defining Reproducible Computational Environments in the Cloud
```

Comment:
Could potentially drop “Defining”. Maybe change “Crowdsourcing” to “Contribute” to make it an action for the reader?

> Contribute community practices for reproducible computational environments in the cloud

Hmm, sounds a bit flat, but maybe we can iterate on this if you like.